app/routers/calendar_grid.py

```diff
@@ -0,0 +1,49 @@
+import calendar
+import itertools
+from datetime import datetime, timedelta
+
+from typing import Any, Iterator, Generator, List
+
+DAYS_OF_THE_WEEK: List[str] = ['Monday', 'Tuesday', 'Wednesday',
```

Comment:
see comments in prev PR and please fix before creating a new PR.

Reply:
Done